diff --git a/cjs/kg-default-nodes.js b/cjs/kg-default-nodes.js
index c7fc87251c73f5ff9e19a17854ea12ccaf45febc..b97a34b8d3e4e682dd74b4a5b573171e2856d143 100644
--- a/cjs/kg-default-nodes.js
+++ b/cjs/kg-default-nodes.js
@@ -974,14 +974,16 @@ class ImageNode extends generateDecoratorNode({
         {name: 'cardWidth', default: 'regular'},
         {name: 'width', default: null},
         {name: 'height', default: null},
-        {name: 'href', default: '', urlType: 'url'}
+        {name: 'href', default: '', urlType: 'url'},
+        {name: 'focalPoint', default: null},
+        {name: 'maxWidthPx', default: null}
     ],
     defaultRenderFn: renderImageNode
 }) {
     /* @override */
     exportJSON() {
         // checks if src is a data string
-        const {src, width, height, title, alt, caption, cardWidth, href} = this;
+        const {src, width, height, title, alt, caption, cardWidth, href, focalPoint, maxWidthPx} = this;
         const isBlob = src && src.startsWith('data:');
 
         const dataset = {
@@ -994,7 +996,9 @@ class ImageNode extends generateDecoratorNode({
             alt,
             caption,
             cardWidth,
-            href
+            href,
+            focalPoint,
+            maxWidthPx
         };
         return dataset;
     }
diff --git a/es/kg-default-nodes.js b/es/kg-default-nodes.js
index 97875fe2bbc9385694681162885dd477a2e96a21..9a9f032d1707cbcc3d46b6a0f36dcf527f8aa971 100644
--- a/es/kg-default-nodes.js
+++ b/es/kg-default-nodes.js
@@ -973,6 +973,12 @@ class ImageNode extends generateDecoratorNode({
     name: 'href',
     default: '',
     urlType: 'url'
+  }, {
+    name: 'focalPoint',
+    default: null
+  }, {
+    name: 'maxWidthPx',
+    default: null
   }],
   defaultRenderFn: renderImageNode
 }) {
@@ -987,7 +993,9 @@ class ImageNode extends generateDecoratorNode({
       alt,
       caption,
       cardWidth,
-      href
+      href,
+      focalPoint,
+      maxWidthPx
     } = this;
     const isBlob = src && src.startsWith('data:');
     const dataset = {
@@ -1000,7 +1008,9 @@ class ImageNode extends generateDecoratorNode({
       alt,
       caption,
       cardWidth,
-      href
+      href,
+      focalPoint,
+      maxWidthPx
     };
     return dataset;
   }
diff --git a/lib/nodes/image/ImageNode.js b/lib/nodes/image/ImageNode.js
index 78dacba20a8eaad1f35364732aa8aa4d69556ab3..42097bcafa39c20de1645ec17aab4c9cee04f060 100644
--- a/lib/nodes/image/ImageNode.js
+++ b/lib/nodes/image/ImageNode.js
@@ -11,14 +11,16 @@ export class ImageNode extends generateDecoratorNode({
         {name: 'cardWidth', default: 'regular'},
         {name: 'width', default: null},
         {name: 'height', default: null},
-        {name: 'href', default: '', urlType: 'url'}
+        {name: 'href', default: '', urlType: 'url'},
+        {name: 'focalPoint', default: null},
+        {name: 'maxWidthPx', default: null}
     ],
     defaultRenderFn: renderImageNode
 }) {
     /* @override */
     exportJSON() {
         // checks if src is a data string
-        const {src, width, height, title, alt, caption, cardWidth, href} = this;
+        const {src, width, height, title, alt, caption, cardWidth, href, focalPoint, maxWidthPx} = this;
         const isBlob = src && src.startsWith('data:');
 
         const dataset = {
@@ -31,7 +33,9 @@ export class ImageNode extends generateDecoratorNode({
             alt,
             caption,
             cardWidth,
-            href
+            href,
+            focalPoint,
+            maxWidthPx
         };
         return dataset;
     }
